README.md

```diff
@@ -320,14 +327,102 @@ const MapContainer = ({dispatch}) => (
 ```
 Read more about [forward dispatching actions][forward-actions]
 
+### 4. Customize style.
+Kepler.gl implements css styling using [Styled-Components](https://www.styled-components.com/). By using said framework Kepler.gl offers the ability to customize its style/theme using the following approaches:
+- Passing a Theme prop
+- Styled-Components ThemeProvider
+
+
+The available properties to customize are listed here [theme](https://github.com/uber/kepler.gl/blob/master/src/styles/base.js).
+
+[Custom theme example](https://github.com/uber/kepler.gl/tree/master/examples/custom-theme).
+
+#### Passing a Theme prop.
+You can customize Kepler.gl theme by passing a __theme__ props to Kepler.gl react component as it follows:
+```javascript
+// Properties are defined in https://github.com/uber/kepler.gl/blob/master/src/styles/base.js
```

Comment:
this is important information, should not be listed in a comment

Reply:
I will remove it.

The same information is listed on line 336 of this file